modules/core/src-2.12/schema/MapTagCompanionPlatform.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright 2021-2025 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s
+package schema
+
+import scala.collection.immutable.ListMap
+
+trait MapTagCompanionPlatform {
+  type SeqMapType[K, V] = ListMap[K, V]
+
+  case object SeqMapTag extends MapTag[ListMap] {
+    override def name: String = "ListMap"
+
+    override def iterator[K, V](c: ListMap[K, V]): Iterator[(K, V)] =
+      c.iterator
+
+    override def build[K, V](
+        put: (((K, V)) => Unit) => Unit
```

Comment:
```suggestion
        put: ((K, V) => Unit) => Unit
```

Does it need those extra parenthesis? Honest question really, not sure

Reply:
With the extra parenthesis it's a function that takes in a single argument that is a tuple. Without the parenthesis, it's a function that takes two arguments. With that in mind though since the use case for this is for other libraries to generate their own map like thing I like the function taking two arguments instead of a single tuple for libraries that are not built in Scala. any opinions?